openam-server-only/src/main/resources/config/validserverconfig.properties

```diff
@@ -338,4 +338,4 @@ org.forgerock.openam.encryption.key.size=128,192,256
 org.forgerock.openam.encryption.key.digest=SHA1,SHA256,SHA384,SHA512
 org.forgerock.openam.radius.server.context.cache.size=integer
 XUI.enable=false,true
-
+org.forgerock.openam.httpclienthandler.system.proxy.enabled=false,true
```

Comment:
org.forgerock -> org.openidentityplatform

Reply:
I left the org.forgerock namespace to remain compatible with the feature documentation at https://backstage.forgerock.com/docs/am/6/install-guide/#JVM-proxy-server . I can change it, but then we'll have to document this difference. What do you think?